collections/Canon_Pat_Lat/MS_Canon_Pat_Lat_52.xml

```diff
@@ -52,173 +45,320 @@
+                        <extent>ii (modern paper) + ii (parchment fragment) + 205 + ii (modern paper) leaves <dimensions unit="mm" type="leaf">
+                              <height>180</height>
+                              <width>130</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Contemporary foliation, ‘1‘–‘45‘ (fols. 1v–43r). Modern foliation doubles several numbers: fols. 5a/5b, 23a/23b, 46a/46b, 67a/67b.</foliation>
+                        <collation>a<hi rend="superscript">two</hi> (fols. iii–iv, attached singletons) | 1–3<hi rend="superscript">12</hi> (fols. 1–34, including 5a/5b, 23a/23b) | 4–6<hi rend="superscript">12</hi> (fols. 35–68, including 46a/46b, 67a/67b) | 7<hi rend="superscript">12</hi> (fols. 69–80), 8–9<hi rend="superscript">10</hi> (fols. 81–100) | 10–15<hi rend="superscript">12</hi> (fols. 101–172) | 16<hi rend="superscript">16</hi> (fols. 173–188) | 17<hi rend="superscript">12–1</hi> (fols. 189–199; 12th cancelled)</collation>
                      </supportDesc>
+                     <layoutDesc>
+                        <layout writtenLines="50 65" columns="2">Ruled in ink, 2 columns of 50–65 lines</layout>
+                     </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                     <handNote script="textualisNorthern">Textura by at least two hands (the second from fol. 000).</handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="colInit">Space left for initials, not executed.</decoNote>
+                  </decoDesc>
+                  <bindingDesc>
+                     <binding notBefore="1700" notAfter="1760">
+                        <p>Soranzo binding: brown tanned calf over pulpboard.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" notAfter="1410" notBefore="1400">15th century, beginning</origDate>
+                     <origDate calendar="Gregorian" notAfter="1400" notBefore="1350">14th century, second half</origDate>
+                     <origPlace key="place_1000080">Italian</origPlace>
                   </origin>
-               <provenance notBefore="1727" notAfter="1805" resp="#MMM"><persName role="fmo" key="person_2384880">Matteo Luigi Canonici, 1727-1805</persName></provenance><provenance notAfter="1807" resp="#MMM"><persName role="fmo" key="person_446">Giuseppe Canonici , -1807</persName></provenance><acquisition when="1817" resp="#MMM">Purchased by the Bodleian in 1817</acquisition></history>
-               <additional><adminInfo>
+                  <provenance>Cistercian monastery in the diocese of Taranto, Lecce: <quote>Statio huius libri est in quarta sede a latere canalis. Iste liber est monasterii caritatis</quote> (fol. 199v; 15th century).</provenance>
```

Comment:
Is this Smalley's identification? if so, it refers to a particular monastery that we should index (https://gallica.bnf.fr/ark:/12148/bpt6k5767414x/f319.image.r=caritatis; there is a more up to date index of Cistercian monasteries in the gallery somewhere as well). Not sure how convincing the identification is, in any case, if only because southern Italy seems a stretch for finding German content (northern Italy less so)

Reply:
I had stopped looking into it further after seeing this: 'Dr. Martin Kaufmann, Assistant Librarian at the Bodleian Library, advises that this probably refers to the Cistercian monastery of S. Maria de Caritate in the diocese of Taranto (province of Lecce)' (*Analecta Cartusiana* 119, p. 74). This writer did not know about the Smalley article. The identification appears to be from an annotation in the reading room copy of the catalogue. Smalley merely cites the source to which you link, which in spite of her 'must' does not provide any evidence.

I was interpreting the hand as northern Italian.

I've now found that Aberdeen University Library MS 242 has an ex libris with [very similar wording](https://babel.hathitrust.org/cgi/pt?id=mdp.39015034710452&view=1up&seq=117) – M.R. James vaguely suggests that it might have been from a house by this name in Milan or Venice (and thinks their MS 106 is from the same place).

I think 'a latere canalis' must refer to Maria della Carità in Venice: this was a house of regular canons (which fits better with its next home in Vicenza) and was located beside the Grand Canal.

Reply:
The authority for this can be taken from VIAF: <https://viaf.org/viaf/238824653/>.

We have an entry for San Bartolomeo, but it's not linked to VIAF: <https://viaf.org/viaf/238618589/>.

Reply:
Actually, Maria della Carità is already linked to MS. Lat. th. d. 28 – and that MS was also at San Bartolomeo!

Reply:
Nice find! Let me know if any new VIAF authorities are needed and when this is ready to go